log-collector-script/linux/eks-log-collector.sh

```diff
@@ -20,14 +20,23 @@ export LANG="C"
 export LC_ALL="C"
 
 # Global options
-readonly PROGRAM_VERSION="0.7.8"
+readonly PROGRAM_VERSION="0.7.9"
 readonly PROGRAM_SOURCE="https://github.com/awslabs/amazon-eks-ami/blob/main/log-collector-script/"
 readonly PROGRAM_NAME="$(basename "$0" .sh)"
 readonly PROGRAM_DIR="/opt/log-collector"
 readonly LOG_DIR="/var/log"
 readonly COLLECT_DIR="/tmp/eks-log-collector"
 readonly CURRENT_TIME=$(date --utc +%Y-%m-%d_%H%M-%Z)
-readonly DAYS_10=$(date -d "-10 days" '+%Y-%m-%d %H:%M')
+
+# Global options for journalctl - https://www.freedesktop.org/software/systemd/man/latest/journalctl.html
+
+# Unify the out output with precise ISO timestamp (e.g. 2025-02-12T06:25:29.149489+0000)
+# It might be needed to have nanosecond timestamp for advanced troubleshooting.
+readonly JOURNALCTL_CMD='/usr/bin/journalctl --output=short-iso-precise'
```

Comment:
EKS node might have customized timezone set, precisely log the timestamp would be very helpful for troubleshooting.